src/Operation/Find.php

```diff
@@ -314,6 +313,7 @@ public function execute(Server $server)
         $cursor = $server->executeQuery($this->databaseName . '.' . $this->collectionName, new Query($this->filter, $this->createQueryOptions()), $this->createExecuteOptions());
 
         if (isset($this->options['codec'])) {
+            /** @var CursorInterface<int, object>&Iterator<int, object> $cursor */
```

Comment:
`$cursor` is assigned a few lines above, so I'm curious why phpcs is unable to find it. Should this warning be suppressed?

Reply:
This has now been solved.